src/api/app/views/webui2/webui/staging/workflows/_problems.html.haml

```diff
@@ -1 +1,31 @@
-Placeholder for the content
+:ruby
+  max_problems = 5
+  project_class = project.name.tr(':', '_')
+  item_project_class = 'd-none hidden-item-' + project_class
+
+- if project.problems.empty?
+  .text-center
+    %i.fas.fa-check-circle.fa-2x.text-success
+- else
+  %ul.list-group.list-group-flush
+    - project.problems.each.with_index(1) do |(name, states), index|
+      %li.list-group-item.table-list-group-item{ class: "#{item_project_class if index > max_problems}" }
+        = link_to("#{name}:", package_show_path(package: name, project: project))
+        - states.each do |state, values|
+          = state
+          = surround '(', ')' do
+            - values.each do |value|
```

Comment:
better sort by architecture. It looks rather silly if it's random order.

Reply:
they come already sort

Reply:
Sorted on what key? Architecture?

Reply:
`values.sort{ |a,b| a[:arch] <=> b[:arch] }.each do |value|` :thinking:

Reply:
@coolo we agreed in the end that we don't do this, right?